CHANGELOG.md

```diff
@@ -48,6 +48,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Downgraded a check to an error in SpatialSender::SendAuthorityIntentUpdate when sending the same intent twice.
 - Fixed a client crash that sometimes occurred when quickly unloading and reloading sublevels.
 - Fixed a worker crash when calling RPCs on PlayerControllers with a certain timing.
+- Fixed a bug where actors with replicated lists, such as TArrays, would semi-rarely not be updated if the list was emptied.
```

Comment:
Should I actually explain when this could have been the case? I feel like this might create undue panic maybe

Reply:
`Fixed a bug where on initial replication, actors with replicated TArrays, would not have the array cleared if the default state was not empty` ?

Reply:
And bNetLoadOnClient actors which go out of view and come back with a cleared TArray.

Reply:
And  upon migration if the default state was not empty and the replicated state is an empty array.

Reply:
Initial replication covers re-entrant replication.
Not sure why we need to call out bNetLoad - it's the same actors who have default states of arrays full.
I'd just go with what I said :)

Reply:
bNetLoadOnClient actors can have the default state be an empty array, have something added to them, the thing moves out of view, gets the array emptied, comes back into view, the array is not updated.

Reply:
I guess we could just say `if the local state was not empty`? That could cover both

Reply:
Ah, and @samiwh, do we still get a component data when gaining authority with SpatialView?
In that case that's not really initial replication.

Reply:
It's not just on initial replication, it will also happen if something leaves and re-enters your view (which is in general is not the same as setting initial values), authority changes, ownership changes, and could happen at other times if we decide there is some reason to refresh state.

Reply:
For a release note I'm not sure it matters that much. If the bug was only seen on initial replication at the moment then that's the thing to communicate we've fixed.

Reply:
I think Mike meant initial replication in Unreal terms, as a client observing the initial bunch (which we don't have, but the approximate equivalent). I thought about it some more, and I think saying "initial" is ok, as it catches the most common cases we want to communicate.